backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHIteratorApi.scala

```diff
@@ -49,12 +49,15 @@ class CHIteratorApi extends IIteratorApi with Logging {
       index: Int,
       partitions: Seq[InputPartition],
       wsCxt: WholestageTransformContext): BaseNativeFilePartition = {
+    logDebug(s"partition number:${partitions.size}")
```

Comment:
these four 'logDebug' are only for dev? will be removed ?

Reply:
removed